### PR TITLE
LPS-161917, LPS-161918, LPS-161919, LPS-161920 - Objects Automated Tests - ObjectNotifications

### PR DESCRIPTION
**Pull request by alanafergs**

### CODE REVIEW

**Comment by alanafergs** [comment]
ci:test:sf

File: modules/apps/object/object-test/src/testFunctional/paths/ObjectAction.path [hunk 1]
```diff
@@ -52,6 +57,11 @@
 		<td>//span[contains(., 'Enable Condition')]/..//span[contains(@class, 'toggle-switch-check-bar')]//input[contains(@type, 'checkbox')]</td>
 		<td></td>
 	</tr>
+	<tr>
+		<td>AUTOCOMPLETE_INPUT</td>
+		<td>//div[@class='dropdown-menu autocomplete-dropdown-menu show']//button[@class='dropdown-item' and contains(., '${key_entryName}')]</td>
+		<td></td>
+	</tr>
```
Comment (timpak) [inline modules/apps/object/object-test/src/testFunctional/paths/ObjectAction.path:64]: This feels a little fragile, especially if the class value changes. We can use a part of the value like `dropdown-menu` or even skip the `div` entirely and just use the button, as long as it doesn't find duplicate elements.

For now, it's fine but in the future please keep this in mind when designing paths. If you're not sure what I mean, schedule a meeting with me and I can explain further.

File: modules/apps/object/object-test/src/testFunctional/tests/ObjectNotifications.testcase [hunk 2]
```diff
@@ -120,4 +120,75 @@ definition {
...
+			ObjectAdmin.fillObjectAction(
+				actionName = "Action Name",
+				notificationName = "Notification Template",
+				thenAction = "Notification",
+				whenAction = "On After Add");
+
+			Button.clickSave();
+		}
+
+		task ("Go to created Object and add a Entry") {
```
Comment (timpak) [inline modules/apps/object/object-test/src/testFunctional/tests/ObjectNotifications.testcase:178]: Go to the created Object and add a Entry

File: modules/apps/object/object-test/src/testFunctional/tests/ObjectNotifications.testcase [hunk 3]
```diff
@@ -120,4 +120,75 @@ definition {
...
+			ObjectAdmin.goToCustomObject(objectName = "CustomObject161920");
+
+			ObjectAdmin.addObjectStringFieldEntryViaUI(entryValue = "Entry");
+		}
+
+		task ("Go to Notifications Queue and assert that notification is present") {
```
Comment (timpak) [inline modules/apps/object/object-test/src/testFunctional/tests/ObjectNotifications.testcase:184]: Go to the Notifications Queue and assert the notification is present

**Review by timpak — COMMENTED** [review]
Good job on this test, @alanafergs! Please see my comments, I'll make the necessary changes myself and forward this. I'll comment later with the PR.

Also, please make sure you use the correct ticket number for each commit. I'll fix it for now but please keep that in mind for next time.

**Comment by timpak** [comment]
Added to https://github.com/liferay-bpm-qa/liferay-portal/pull/9

**Pull request closed without merging**